smarts/core/smarts.py

```diff
@@ -424,11 +449,9 @@ def destroy(self):
 
         super().destroy()
 
-    def _teardown_agent_vehicles(self, agent_ids):
-        torndown_vehicles = self._vehicle_index.teardown_vehicles_by_actor_ids(
-            agent_ids
-        )
-        self._clear_collisions(agent_ids)
+    def _teardown_agent_vehicles(self, vehicle_ids):
```

Comment:
I was confused about the use of this method when reading above. It does not seem to have anything to do with agents and just with vehicles perhaps something like `_teardown_vehicles_by_ids()` or `_teardown_vehicles()`?

Reply:
Good point, I updated it in #203